config/settings.yml

```diff
@@ -34,3 +34,4 @@ tweet_messages:
   - 'Serviço no Brasil é sempre ruim, mas o do @santander_br consegue ser o pior'
   - 'Precisa ligar pro @santander_br pra resolver algo? Se prepare pra esperar 20 minutos pra ser atendido'
   - 'Aquela promessa de resolver tudo pelo app do @santander_br é só promessa mesmo...'
+google_analytics: UA-108690658-1
```

Comment:
I'm not sure if we should leave a code there, cause if the user doesnt change it, his metrics will be sent to you =s

What was your intention here? Just showing an example? 🤔

Reply:
That's a good point. Yes I was showing an example. Ok! I'll remove the tag and keep it empty.